docs/api.md

```diff
@@ -526,3 +527,66 @@ coll.use(new TitleCharTransformer("?"));
 There's currently no way to deal with adding tranformers to an already filled remote database; that would mean remote data migrations, and both Kinto and Kinto.js don't provide this feature just yet.
 
 **As a rule of thumb, you should only start using transformers on an empty remote collection.**
+
+## Id schemas
+
+By default, kinto.js uses UUID4 strings for record id's. If you want to work with an existing body of data, this may not be what you want.
+
+You can define a custom id schema on a collection by calling the `Collection#use()` method, which accepts a `Kinto.IdSchema`-derived object instance:
+
+```js
+import Kinto from "kinto";
+
+class IntegerIdSchema extends IdSchema {
+  constructor() {
+    super();
+    this._next = 0;
+  }
+  generate() {
+    return this._next++;
+  }
+  validate(id) {
+    return ((id == parseInt(id, 10)) && (id >= 0));
```

Comment:
Yes I like that, plus a note saying that:

    In a real application, you want to make sure you do not generate twice the same record id on a collection.
    This dummy example doesn't take care of id unicity. In case of id conflict you may loose data.